cds_ils/migrator/json_record_loader.py

```diff
@@ -47,6 +52,20 @@ def create_record(
                     object_type="rec",
                     object_uuid=record_uuid,
                 )
+
+                config = current_app.config
```

Comment:
not a blocker: I would put these lines in a separate function (the mapping) in order to keep the `create_record` method's code cleaner